rust/migration/src/m20220101_000001_create_blob_tables.rs

```diff
@@ -35,6 +35,7 @@ impl MigrationTrait for Migration {
                     )
                     .col(ColumnDef::new(Blob::Key).string().not_null())
                     .col(ColumnDef::new(Blob::StoreId).uuid().not_null())
+                    .col(ColumnDef::new(Blob::Size).big_integer().not_null())
```

Comment:
Does this really align with the `size: i64` in the model?
I would have assume big_integer was designed to be something potentially much larger.

Reply:
It does. The largest datatype supported by Postgres is `i64`. I actually wanted a `u64` but its not supported. 

Also good to remember that the `entity` structs are generated from the actual database schema so if it was a different type then it would have been generated differently

aside: i64::MAX is still something like 1888 Petabytes which we will hopefully never have a single file that large

Reply:
Yeah not concerned about the actual size I agree its plenty large enough with or without the extra power of two :)